pkgs/by-name/lx/lxgw-wenkai-screen/package.nix

```diff
@@ -8,16 +8,24 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "lxgw-wenkai-screen";
   version = "1.520";
 
-  src = fetchurl {
-    url = "https://github.com/lxgw/LxgwWenKai-Screen/releases/download/v${finalAttrs.version}/LXGWWenKaiScreen.ttf";
-    hash = "sha256-AxTlH8AZBu66WNbjv45cAuFRrItUeElEvyorEkXKQe4=";
-  };
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/lxgw/LxgwWenKai-Screen/releases/download/v${finalAttrs.version}/LXGWWenKaiScreen.ttf";
+      hash = "sha256-AxTlH8AZBu66WNbjv45cAuFRrItUeElEvyorEkXKQe4=";
+    })
+    (fetchurl {
+      url = "https://github.com/lxgw/LxgwWenKai-Screen/releases/download/v${finalAttrs.version}/LXGWWenKaiMonoScreen.ttf";
+      hash = "sha256-hyud3lwGi2W45bnz6qz1ovZuvR3bg6xhDdgXXIVjwsI=";
+    })
+  ];
 
   dontUnpack = true;
 
   installPhase = ''
     runHook preInstall
-    install -Dm644 "$src" "$out/share/fonts/truetype/LXGWWenKaiScreen.ttf"
+    for ttf in $srcs; do
```

Comment:
I suggest just spell out the font names instead of using a for loop.

Reply:
> Then we are at a higher risk of missing some newly added fonts in updates.

This is exactly what I prefer hard coding font names. I don't like changes from upstream causing unsupervised/unscrutinized change on NixOS's side.